internal/pkg/plugin/devlakeconfig/devlakeconfig.go

```diff
@@ -99,7 +178,66 @@ func DeleteConfig(options plugininstaller.RawOptions) error {
 }
 
 func UpdateConfig(options plugininstaller.RawOptions) error {
-	// TODO(daniel-hutao): implement later
+	opts, err := NewOptions(options)
+	if err != nil {
+		return err
+	}
+	cfg, err := configmanager.NewManager("config-devlake-config.yaml").LoadConfig()
```

Comment:
You can get the `Config` from options(function param). ConfigManager is used in `core` only for the "config file name" is only known by core. It's not the "config-devlake-config.yaml".